newIDE/app/src/AssetStore/ExtensionStore/ExtensionsSearchDialog.js

```diff
@@ -19,7 +19,11 @@ type Props = {|
 /**
  * Allows to browse and install events based extensions.
  */
-export default function ExtensionsSearchDialog({ project, onClose }: Props) {
+export default function ExtensionsSearchDialog({
+  project,
+  onClose,
+  onCloseEventsFunctionsExtensionTab,
```

Comment:
While this works, this gives too much responsibility on the ExtensionSearchDialog, which must find the eventsFunctionExtension associated to the installed extension. This component should not care about this (You can picture it telling: "I'm a search and install dialog, I don't care about your freaking tabs" ;))

Instead, let's call this prop "onInstallExtension", getting a ExtensionShortHeader as parameter.